src/TestingComponents/SignIn.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import SignIn from 'pages/SignIn';
+import store from '../store';
+
+test('renders "Sign In" as a text', () => {
+  render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <SignIn />
+      </BrowserRouter>
+    </Provider>,
+  );
+  const searchElement = screen.getByText('Sign In');
+  expect(searchElement).toBeInTheDocument();
+});
+test('should the button have attribute submit', () => {
```

Comment:
Same note as above; this doesn't technically verify the button has an attribute submit and only verifies it is on the page.